### DISCUSSION TITLE
Getting delayed video

### DISCUSSION THREAD

**Discussion by madgiordano** [gopro/labs#1111] [Q&A]
Hi!

I've had great success with scheduling photos previously and now I'm trying to schedule video. Because battery is so limited when taking video, I am trying to program in a start date. I would really love to schedule a start date, let's say Oct15 and have the camera take 20 minutes of video after that at 6-620 and 7-720 for example, every day until the battery dies. But I can't get it to recognize a date as a date and not a time. I was trying to use the action code I found, something like >18:00:20241015 - and then add the rest of the code. Maybe I'm misunderstanding how to write a date, or maybe starting after a date and time isn't possible? I have been generally unsuccessful. Maybe I can program all screens off to maintain battery if I can't get it to wait to start the schedule. (The camera will be underwater for a month so screens are unimportant)

Has anyone had any luck with repeating daily schedules after a period of rest?
 
Thanks so much for any advice!

**Comment by dnewman-gpsw** [comment]
The original delay start feature for Labs used on this rocket launch https://www.instagram.com/p/CAcBXNflYwn/ where the camera had to placed more than 72 hours in advance. So you wanted exactly 72 hours the command is !259200S, you specify the delay in seconds.  This old feature seems broken (working it that.) However, there is a more precise way with newer cameras.

t:D variable in the Day of the date.
t:M variable in the Month of the date.

If you want the camera to start on the 15th of any month do something like:
```<t:D15!05:59R!06:00S!1200E!07:00S!1200E!R13```

```
<t:D15   - if(dataDay < 15)
!05:59R  -        sleep until the next 5:59am, repeat from top
!06:00S  - start at 6am in the default capture mode
!1200E   - end after twenty minutes
!06:00S  - sleep and start at 7am in the default capture mode
!1200E   - end after twenty minutes
!R13       - repeat at character position 13 (!06:00S)
```
